src/components/molecules/atoms/TabItem.tsx

```diff
@@ -8,20 +8,21 @@ interface IProps {
   onClick: (index: number) => void;
 }
 
-const TabItem: React.FC<IProps> = (props) => {
-  const {className} = props;
+const LI = styled.li`
+  display: inline-block;
+  list-style: none;
+  margin-bottom: -1px;
+  padding: 0.5rem 0.75rem;
+`;
 
+const TabItem: React.FC<IProps> = (props) => {
   const handleClick = () => props.onClick(props.tabIndex);
   return (
-    <li onClick={handleClick} className={className}>{props.children}</li>
+    <LI onClick={handleClick}>{props.children}</LI>
   );
 };
 
 export default styled(TabItem)`
-  display: inline-block;
-  list-style: none;
-  margin-bottom: -1px;
-  padding: 0.5rem 0.75rem;
   ${props => props.isActive ? `border: solid #ccc;
```

Comment:
For a simpler ternary expression (the `comparison ? true : false` syntax) without the `else` case, you can use `{props.isActive && \`...\`}`. No difference, but a bit more intuitive imo.